src/Umbraco.Web/PropertyEditors/NestedContentPropertyEditor.cs

```diff
@@ -266,6 +266,45 @@ public override object FromEditor(ContentPropertyData editorValue, object curren
                 return JsonConvert.SerializeObject(value);
             }
 
+            public IEnumerable<UmbracoEntityReference> GetReferences(object value)
+            {
+                var rawJson = value == null ? string.Empty : value is string str ? str : value.ToString();
+
+                var result = new List<UmbracoEntityReference>();
+
+                var list = JsonConvert.DeserializeObject<List<object>>(rawJson);
```

Comment:
Would it be better to just Deserialize as `List<JObject>` then you don't have to cast below